### ISSUE TITLE
Cookie list appears blank

### ISSUE THREAD

**Issue by Marek-1** [mrdokenny/Cookie-AutoDelete#563]
After grey/whitelisting a domain and unchecking "keep all cookies", the list expands, but no cookies appear

![cad bug — kopia](https://user-images.githubusercontent.com/47012223/63597195-30385600-c5bd-11e9-8100-a1e7a7135053.png)

STR
greylist a domain through pop-up
uncheck "keep all cookies"

- OS/version: Windows 10
- Browser/version: Firefox 68.0.2 64bit
- Cookie AutoDelete version: 3.0.2

**Comment by kennethtran93** [comment]
Perhaps the cookies were deleted already as per the recently deleted cookies line right below that mentioning github.com.  Did you try restoring the cookies with the undo button right beside that log line mentioning github.com?

**Comment by Marek-1** [comment]
I don't think so. I re-opened browser and opened new tabs, there was the same situation. I also reinstalled the addon before this.

![bug2](https://user-images.githubusercontent.com/47012223/63649851-3b1cf300-c743-11e9-9d78-bbf020728eea.png)
![bug1](https://user-images.githubusercontent.com/47012223/63649852-3bb58980-c743-11e9-988c-c21e57732f2b.png)

**Comment by kennethtran93** [comment]
From my experience the cookie names would only show up if the cookies still exist.

- Are you using private browsing or permanent private browsing?
- Any containers in use?


In the same browser session where the popup shows no individual cookies...
- Are the cookies still shown in Firefox's internal cookie window?
- If you used a cookie manager extension and searched up the cookies do you still see them?

**Comment by kennethtran93** [comment]
![(50) YouTube - Firefox Developer Edition](https://user-images.githubusercontent.com/6724477/63732313-82c97a80-c828-11e9-80d1-3c506292f06a.png)

I'll definitely need more details to replicate what you are seeing.  I'm using Firefox Developer Edition which is on 69.0b16 and it's not resulting on what you had experienced (and I did that twice).  I checked upon first load of the Youtube site and it shows ~4-5 cookies being stored.  After it found other Google services through some means then it redirected to my logged in profile.
